commands/core/config.drush.inc

```diff
@@ -146,6 +156,31 @@ function config_drush_command() {
       'drush --bg config-edit image.style.large' => 'Return to shell prompt as soon as the editor window opens.',
     ),
   );
+  $items['config-merge'] = array(
+    'description' => 'Combine configuration data from this site with the configuration data of another site; if there are conflicts, open an interactive three-way merge tool comparing the changes with the base revision.',
+    'aliases' => array('cm'),
+    'required-arguments' => 1,
+    'arguments' => array(
+      'site' => 'Alias for the site containing the other configuration data to merge.',
+      'label' => "A config directory label (i.e. a key in \$config_directories array in settings.php). Defaults to 'staging'",
+    ),
+    'options' => array(
+      'base' => 'The commit hash or tag for the base of the three-way merge operation.  This should be the most recent commit that was deployed to the site specified in the first argument.',
+      'tracking' => 'A tracking branch to use when doing the configuration merge (advanced). Default is to use a temporary branch.',
+      'comment' => 'Commit comment for the merged configuration.',
```

Comment:
I think the word 'message' is more common than 'comment' when referring to commit text.